EventListener/ReferrerPolicyListener.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Nelmio SecurityBundle.
+ *
+ * (c) Nelmio <hello@nelm.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nelmio\SecurityBundle\EventListener;
+
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+/**
+ * Referrer Policy Listener.
+ *
+ * @author Andrej Hudec <pulzarraider@gmail.com>
+ */
+class ReferrerPolicyListener
+{
+    private $policies;
+
+    public function __construct(array $policies)
+    {
+        $this->policies = $policies;
+    }
+
+    public function onKernelResponse(FilterResponseEvent $e)
+    {
+        if (HttpKernelInterface::MASTER_REQUEST !== $e->getRequestType()) {
+            return;
+        }
+
+        $response = $e->getResponse();
+
+        $response->headers->set('Referrer-Policy', implode(', ', $this->policies));
```

Comment:
adding multiple values looks weird to me. MDN does not seem to imply that putting multiple values is possible.

Reply:
And the W3C spec does not allow it either.

Reply:
From https://www.w3.org/TR/referrer-policy/#unknown-policy-values:

> As described in §8.1 Parse a referrer policy from a Referrer-Policy header and in the meta referrer algorithm, unknown policy values will be ignored, and when multiple sources specify a referrer policy, the value of the latest one will be used. This makes it possible to deploy new policy values.

> Suppose older user agents don’t understand the "unsafe-url" policy. A site can specify an "origin" policy followed by an "unsafe-url" policy: older user agents will ignore the unknown "unsafe-url" value and use "origin", while newer user agents will use "unsafe-url" because it is the last to be processed.

Reply:
Not all browsers support all policies. You can define multiple values for BC with older browsers.